django_server/.env.sample

```
@@ -0,0 +1,80 @@
+
+# Redis Configuration (for Celery and Channels)
+REDIS_URL=redis://redis:6379/0
+CELERY_BROKER_URL=redis://redis:6379/0
+CELERY_RESULT_BACKEND=redis://redis:6379/0
+
+# CORS Configuration
+CORS_ALLOWED_ORIGINS=http://localhost:3000,http://localhost:3001
+
+# JWT Configuration
+JWT_SECRET_KEY=your-jwt-secret-key-here
+JWT_ACCESS_TOKEN_LIFETIME=60  # minutes
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_ACCESS_TOKEN_LIFETIME key should go before the JWT_SECRET_KEY key

(UnorderedKey)